packages/web/README.md

```diff
@@ -14,91 +14,76 @@ React components for web development inside Natura.
 
 ## Getting started
 
-Install and get started with Natura Design System components for React for web applications.
-
 ### Installation
 
-Design System components for React is available as an [npm package](https://www.npmjs.com/package/@naturacosmeticos/natds-web).
+```shell script
+// with npm
+npm install @naturacosmeticos/natds-web
+
+// with yarn
+yarn add @naturacosmeticos/natds-web
+```
 
-#### Stable channel v0
+### Usage
+#### Theme Setup
 
-To get the latest stable release always use `latest` distribution channel.
+This package only works fine with the use of `<Provider />`, that is **essential** for applying styles correctly.
+To avoid problems with multiple style fonts, it is necessary to add a `cssPrefix` with theme provider.
```

Comment:
eu mudaria essa frase pra algo como:
If you have problems with class conflicts or if you ar loading more than one instance of the Design System Library, you can add more specificity by using the `cssPrefix` provider prop.
Se pa isso poderia ficar com menor destaque tbm, pq é mais corner case